tests/unit/s2n_client_ecc_point_format_extension_test.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include "s2n_test.h"
+
+#include "tls/extensions/s2n_client_ec_point_format.h"
+#include "tls/s2n_resume.h"
+
+int main(int argc, char **argv)
+{
+    BEGIN_TEST();
+
+    struct s2n_config *config;
+    EXPECT_NOT_NULL(config = s2n_config_new());
+
+    /* Test send */
```

Comment:
Can we add tests for s2n_client_ec_point_format_extension.should_send() as well?

Reply:
should_send is covered, just not here. ec_point_format doesn't define a unique should_send, it just uses s2n_extension_should_send_if_ecc_enabled (defined in s2n_client_supported_groups)